### PR TITLE
About us

### PR DESCRIPTION
**Pull request by sirinebettaieb**
CREATE MEDIA QUERIES

### CODE REVIEW

**Review by Fatimata-dev — APPROVED** [review]

File: src/app/pages/home/home.component.html [hunk 1]
```diff
@@ -1,2 +1,4 @@
-
-  <app-nav-bar></app-nav-bar>
+<section>
+ <a routerLink="/teams"><span>Ce lien</span></a>
```
Comment (Benja-san) [inline src/app/pages/home/home.component.html]: ce lien ? name it as the page you refer to like team or about us

File: src/app/components/team/team.component.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<article>
+    <img src="assets/pictures/{{ team.picture }}" alt="{{ team.name }}" />
+    <h3 >{{ team.name }}</h3>
```
Comment (Benja-san) [inline src/app/components/team/team.component.html]: no h2 and no h1 on the page, consider using some into the about us page component
OR
change those h3 into h2 and add a h1 into about us component

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/pages/home/home.component.html (hunk 1 above)
Comment (sirinebettaieb) [reply src/app/pages/home/home.component.html]: about us

File: src/app/components/team/team.component.html (hunk 2 above)
Comment (sirinebettaieb) [reply src/app/components/team/team.component.html]: ok

**Review by Fatimata-dev — APPROVED** [review]

File: src/app/components/team/team.component.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <article>
     <img src="assets/pictures/{{ team.picture }}" alt="{{ team.name }}" />
-    <h3 >{{ team.name }}</h3>
-    <h3>E-mail :</h3>
-    <h3>{{ team.email }}</h3>
+    <h1 >{{ team.name }}</h1>
```
Comment (Benja-san) [inline src/app/components/team/team.component.html]: Consider the following example as a good practice for html pages : 
ONE H1 for the main title of the page
as much h2 as you want (till there is one H1)
as much h3 as you want (till there is t least one h2)
and so on.....

In this case use h2 for the team name but email is not a title

And use only one H1 in the about us page

File: src/app/pages/home/home.component.html [hunk 4]
```diff
@@ -1,4 +1,2 @@
-<section>
- <a routerLink="/teams"><span>Ce lien</span></a>
```
Comment (Benja-san) [inline src/app/pages/home/home.component.html]: change "ce lien" by a more explicit sentence

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Fatimata-dev — APPROVED** [review]

File: src/app/components/team/team.component.html [hunk 5]
```diff
@@ -1,8 +1,11 @@
 <article>
     <img src="assets/pictures/{{ team.picture }}" alt="{{ team.name }}" />
-    <h1 >{{ team.name }}</h1>
-    <h2>E-mail :</h2>
-    <h2>{{ team.email }}</h2>
+    <h1>{{ team.name }}</h1>
```
Comment (Benja-san) [inline src/app/components/team/team.component.html]: change h1 for h2 (only one h1 by page)

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**